source/api-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _rm-rest-api:
```

Comment:
**Note to Internal Review**

This is the only real page that needs to be reviewed, the spec file itself is maintained in another repo and should not be considered part of this review.